tests/system/node/0/config.yaml

```diff
@@ -36,6 +36,7 @@ network:
   - http://node-4:4826
   - http://node-5:5826
   - http://node-6:6826
+  - http://node-7:7826
```

Comment:
Node 0 is not a validator, so it doesn't need to know the full set of nodes. That's why it had a lower number of listeners and was missing node 7. Maybe I should have added a comment.

Reply:
But the Validator's configs have the minimum set to 6. So they need to connect to the FullNode too.

Reply:
I think maybe the right fix is to lower the minimum set in the Validator configs.

And actually it's a little dangerous to have this value be applied to the validators because if it's too high the validator will never start validating.